### PR TITLE
Change xml name

### PR DESCRIPTION
**Pull request by mnbp1371**

### CODE REVIEW

**Comment by mnbp1371** [comment]
@vispamir

File: idpay_virtuemart/idpay_virtuemart.xml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <extension version="3.6" type="plugin" group="vmpayment" method="upgrade">
-    <name>IDPay for Virtuemart</name>
+    <name>System - IDPay for Virtuemart</name>
     <version>2.0.0</version>
     <description>IDPay payment gateway for Virtuemart</description>
     <author>JMDMahdi, Vispa</author>
```
Comment (vispamir) [inline idpay_virtuemart/idpay_virtuemart.xml]: Add your name as developer:

Developer: JMDMahdi, vispa, mnbp1371 , Publisher: IDPay

Comment (vispamir) [reply idpay_virtuemart/idpay_virtuemart.xml]: Add Author mail to xml

**Review by vispamir — CHANGES_REQUESTED** [review]

**Comment by mnbp1371** [comment]
Add author name and email in xml file

@vispamir

**Comment by mnbp1371** [comment]
Change plugin name tag in xml file
@vispamir

**Pull request merged**